src/components/Home/BestSelling.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./css/bestselling.css";
+import { Link } from "react-router-dom";
+const BestSelling = (props) => {
```

Comment:
You could also destructure all the props that you are getting here so that you don't need to write `props.something` everywhere.

e.g.
```jsx
const BestSelling = ({imgSrc, hoverimg}) => {
...
```

Reply:
Thanks , i will do that in next pr